themes/default/content/blog/kubernetes-server-side-apply/index.md

```diff
@@ -0,0 +1,738 @@
+---
+title: "Manage Shared Kubernetes Resources Safely with Pulumi"
+
+date: 2022-07-19T17:13:34-06:00
+
+meta_desc: Manage shared Kubernetes resources safely with our new Server-Side Apply support
+meta_image: meta.png
+authors:
+    - levi-blackstone
+tags:
+    - kubernetes
+---
+
+Kubernetes resources often have more than one controller making changes to them. These controllers can include `kubectl`, the Kubernetes control plane, custom operators, or infrastructure as code (IaC) tools like Pulumi.
+With the [v3.20.1 release](https://github.com/pulumi/pulumi-kubernetes/releases/tag/v3.20.1) of the Kubernetes provider, you have some powerful new options for managing shared resources in Kubernetes. In this post, we show you
+how Pulumi can help you work with shared resources safely and effectively.
```

Comment:
Do we need to add somewhere that alll resources now have a "Patch" equivalent so that people know that it's not just the scenarios below?

Reply:
Yeah, it probably makes sense to summarize the improvements up front here. I wrote this as a companion piece to the how-to guide, which covers the technical changes more directly. I focused the blog on a few practical usage examples.